cmd/secrets-provider/main.go

```diff
@@ -51,51 +52,55 @@ func main() {
 		printErrorAndExit(messages.CSPFK009E)
 	}
 
-	// Configure exponential backoff
-	expBackoff := backoff.NewExponentialBackOff()
-	expBackoff.InitialInterval = 2 * time.Second
-	expBackoff.RandomizationFactor = 0.5
-	expBackoff.Multiplier = 2
-	expBackoff.MaxInterval = 15 * time.Second
-	expBackoff.MaxElapsedTime = 2 * time.Minute
+	syncIntervalTime := 3
+	if envSyncIntervalVal := os.Getenv("SYNC_INTERVAL_TIME"); envSyncIntervalVal != "3" && envSyncIntervalVal != "" {
+		syncIntervalTime, _ = strconv.Atoi(envSyncIntervalVal)
+	}
+
+	retryCount := 3
+	if envRetryCount := os.Getenv("RETRY_COUNT"); envRetryCount != "3" && envRetryCount != "" {
+		retryCount, _ = strconv.Atoi(envRetryCount)
+	}
```

Comment:
same as above. constant + read env in other file

Reply:
they are defaults not constants

Reply:
i meant that we add a constant instead of having the `3` spreading around the code in case the default is changed. you can do this in the config file

Reply:
const list will now look like
```
const (
	K8S            = "k8s_secrets"
	CONJUR_MAP_KEY = "conjur-map"
	DEFAULT_RETRY_COUNT = 3
	DEFAULT_SYNC_INTERVAL = 3
)
```

Reply:
you dont need comparison with "3" here either